public_html/wp-content/mu-plugins/blocks/assets/src/organizers/edit.js

```diff
@@ -55,18 +57,12 @@ class OrganizersEdit extends Component {
 			path: addQueryArgs( '/wp/v2/organizer_team', ALL_TERMS_QUERY ),
 		} );
 
-		if ( this.isStillMounted ) {
-			this.setState( {
-				allOrganizerPosts : allOrganizerPosts, // Promise
-				allOrganizerTerms : allOrganizerTerms, // Promise
-			} );
+		this.state = {
```

Comment:
Yup, in this case, these methods are actually parts of constructors, so makes sense to initialize states this way. As in they are called synchronously from constructor.

Whenever its not synchronous, I am `setState` instead.

Reply:
Ah, I see, makes sense.

Reply:
`[SHOULD]` - Since it is in a separate function and not the constructor, it might be worth adding a comment to the `this.state =` line, indicating that it's safe b/c it's called from the constructor, and warning our future selves to never call that function from outside the constructor.

Reply:
Actually, the linter complains about this, so unless there's some big downside, I think we should just use `setState` in called functions, even though we don't technically have to.